src/math/Triangle.js

```diff
@@ -103,7 +103,12 @@ class Triangle {
 
 	static getInterpolation( point, p1, p2, p3, v1, v2, v3, target ) {
 
-		this.getBarycoord( point, p1, p2, p3, _v3 );
+		if ( this.getBarycoord( point, p1, p2, p3, _v3 ) === null ) {
+
+			target.set( 0, 0, 0 );
+			return null;
```

Comment:
`getInterpolation()` is used in context of raycasting determine uv coordinates and normals (in `Mesh` and `Sprite`). I think we have to handle the case now when this method returns null and apply other default values instead.

Reply:
> What would a better default be here? Previously it would have just been returning incorrect values.

At least I would return the data type that is expected in the intersection object. Probably with initial numbers or maybe `NaN`. But if there can't be an intersection anyway, it probably doesn't matter.

The issue is that we silently ignored such issues so far. When we now start making methods more strict, we have to think about how we want to handle edge cases.

Reply:
I say we leave it as potentially setting null for the raycast results for now - so it will be obvious if something isn't working. And we can reevaluate if it turns out people are getting null results.

> BTW: Is it possible getInterpolation() could be used with an instance of Vector4? If so, target.set( 0, 0, 0 ); isn't safe.

You're right! Just made the change.